build-system/compile/json-schema/index.js

```diff
@@ -182,42 +182,209 @@ function transformAjvCode(code, scope, config) {
   const findProperty = (properties, name) =>
     properties.find((node) => isObjectProperty(node, name));
 
+  /**
+   * @param {babel.NodePath<babel.types.MemberExpression>} memberExpression
+   * @return {null | undefined | number | string}
+   */
+  function evaluatePropertyKey(memberExpression) {
+    const property = memberExpression.get('property');
+    return property.isIdentifier()
```

Comment:
`foo[bar]`, the `bar` property here is an identifier.

```suggestion
    return !memberExpression.node.computed
```